gitbook-docs/market/economics.md

```diff
@@ -1,124 +1,183 @@
+_c_% of the request cost goes as the commission, but not less than _mc_ tokens.
+It's split between two parties:
 
 * _app\_dev_ part goes to the Proof Market development team;
-* _cir\_dev_ part goes to the Circuit Developer of the ordered statement.
+* _cir\_dev_ part goes to the circuit developer of the ordered statement.
 
-#### Funds withdraw
+### Funds withdrawal
 
-**From the proof requester's perspective:**
+#### From the proof requester's perspective
 
-Each time a new bid is added, an amount equal to the bid cost is locked on the account. The locked amount is debited from the account if the bid is successfully closed. If the bid is not closed within the available time, the customer has the opportunity to withdraw the bid. If the bid is successfully withdrawn, the locked amount will be unlocked.
+Each time a new request is added, an amount equal to the request cost is locked on the account.
+The locked funds are debited from the account once the request is successfully closed.
+If the request is not completed within the specified time interval, the proof requester
+can withdraw the request.
+If the request is successfully withdrawn, the locked funds will be unlocked.
 
-**From the proof producer perspective:**
+#### From the proof producer's perspective
 
-At any moment, a proof producer is able to withdraw funds that are not part of the deposit for their active asks. This means that at each moment, the proof producer has pf∗S locked funds where S is the sum of all their ask orders that are not completed or withdrawn.
+At any moment, a proof producer can withdraw funds that are not part of the deposit
+for their active proposals.
+This means that the proof producer has _pf∗S_ locked funds where _S_ is the sum of all
+their proposal orders that are not completed or withdrawn.
 
-### Ratings
+## Ratings
 
-In the system, each proof producer has a rating. Rating is taken into account in many different processes, for example, in matching, ask placement, etc. For simplicity, the conscientious fulfilment of obligations increases the rating. It increases the likelihood of a bid matching, while the producer’s dishonest work can lead to decreased ratings and various restrictions.
+Each proof producer has a rating in the system.
+Rating impacts many processes, such as matching, proposal placement, and others.
+For simplicity, the conscientious fulfillment of obligations increases the rating.
+It increases the likelihood of a request matching, while the producer's dishonest work
+can lead to decreased ratings and various restrictions.
 
-**The rating System is in the design/development stage; more details on it will be provided soon**
+{% hint style="info" %}
+The rating system is in the design/development stage; more details on it will be provided soon**
+{% endhint %}
 
-### What happens if the proof is not submitted on time
+## What happens if the proof is not submitted on time
```

Comment:
This section has the same topic as "Penalties", so they should be merged. Let's discuss how to do it.